obj/tasks.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    //DOM elements
+    createTaskButton : ".btn",
+    createTaskFormTitle : '.tile__title',
+    createTaskName : 'input[name = "taskDetailsForm.name"]',
+    createTaskDescription : 'textArea[name = "taskDetailsForm.description"]',
+    createTaskBillToClient : '.Select-value',
+    createTaskBillToClientSelectList: '[aria-label="?"]',
+    createTaskHourlyRate : 'input[name = "taskDetailsForm.rate"]',
+    createTaskSaveButton : 'button[type = "submit"]',
+    tasksNameFilter : '//*[@id="borderLayout_eGridPanel"]/div[1]/div/div/div[1]/div[3]/div/div[2]/div[1]/div/div/div/input',
+    tasksNameFilterIcon : '//*[@id="borderLayout_eGridPanel"]/div[1]/div/div/div[1]/div[3]/div/div[1]/div[1]/div[2]/div/span[2]/span',
```

Comment:
Please shorten xpath selectors. I tried randomly cut major part of selector and test passed. I believe selectors could be shorter.